jdaviz/app.py

```diff
@@ -960,11 +1027,11 @@ def add_data_to_viewer(self, viewer_reference, data_path,
             viewer_item = self._viewer_item_by_id(viewer_reference)
         if viewer_item is None:
             raise ValueError(f"Could not identify viewer with reference {viewer_reference}")
-        data_label = self._build_data_label(data_path, ext=ext)
+        data_label = self.return_data_label(data_path, ext=ext, check_unique=False)
         data_id = self._data_id_from_label(data_label)
 
         if clear_other_data:
-            self._update_selected_data_items(viewer_item.get('id'), {})
+            self._update_selected_data_items(viewer_item['id'], {})
```

Comment:
This might be rebase error after #1681 is merged but @bmorris3 should confirm.
```suggestion
            self._update_selected_data_items(viewer_item.get('id'), {})
```